### PR TITLE
Update quit-applications extension

### PR DESCRIPTION
**Pull request by boxpositron**
## Description

- Added force quit action ( It sends a kill -9 signal to the PID of the application )
- Added prompts before action confirmation ( All actions are destructive, so a confirmation improves UX )
- Improved load application method

## Screencast

output

## Checklist

- [x] I read the [extension guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store)
- [x] I read the [documentation about publishing](https://developers.raycast.com/basics/publish-an-extension)
- [x] I ran `npm run build` and [tested this distribution build in Raycast](https://developers.raycast.com/basics/prepare-an-extension-for-store#metadata-and-configuration)
- [x] I checked that files in the `assets` folder are used by the extension itself
- [x] I checked that assets used by the `README` are placed outside of the `metadata` folder

### CODE REVIEW

File: extensions/quit-applications/src/utils/widget/index.ts [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+  return 0;
+}
+
+export function fetchApplicationIcon<T extends Omit<Application, "icon">>(application: T): Application {
+  const [applicationPath] = application.path.split("Contents");
+
+  if (applicationPath.includes(".app")) {
+    return {
+      ...application,
+      fileIcon: applicationPath,
```
Comment (mackopes) [inline extensions/quit-applications/src/utils/widget/index.ts:16]: Is it enough to set the icon to the path of the app?

File: extensions/quit-applications/src/utils/widget/index.ts [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+//   }
+
+//   // if icon doesn't end with .icns, add it
+//   if (!iconFileName.endsWith(".icns")) {
+//     iconFileName = `${iconFileName}.icns`;
+//   }
+
+//   const iconPath = `${path}/Contents/Resources/${iconFileName}`;
+//   console.log(iconPath);
+//   return iconPath;
```
Comment (mackopes) [inline extensions/quit-applications/src/utils/widget/index.ts:65]: If this code is not useful anymore, shall we remove it?

Comment (boxpositron) [reply extensions/quit-applications/src/utils/widget/index.ts:65]: Sure I can remove it.

**Comment by boxpositron** [comment]
Alright @mackopes

I agree with your feedback on the confirmation prompt; I'll implement that. 
I can make out time to split the PR's during the week.

**Comment by pernielsentikaer** [comment]
Is this ready or not yet?

**Comment by mackopes** [comment]
I believe not @pernielsentikaer,

@boxpositron I am happy to give it another review once you remove the confirmation and split the PR into two smaller ones, one with the refactor and one with new functionality.

**Pull request closed without merging**